webpack/assets/javascripts/react_app/components/factChart/factChartModal/index.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Button, Modal } from 'react-bootstrap';
+import PieChart from '../../common/charts/PieChart/';
+import helpers from '../../../common/helpers';
+import * as FactChartActions from '../../../redux/actions/factCharts/';
+import Loader from '../../common/Loader';
+import MessageBox from '../../common/MessageBox';
+
+class FactChartModal extends React.Component {
+  constructor(props) {
+    super(props);
+    helpers.bindMethods(this, ['closeModal']);
+  }
+
+  closeModal() {
+    this.props.closeModal();
+  }
+
+  render() {
+    const { factChart } = this.props;
+
+    const chart = <PieChart data={factChart.chartData} key={`pie-chart-${factChart.id}`}/>;
+
+    const error = (
+      <MessageBox
+        msg={__('No data available')}
+        icontype="error-circle-o"
```

Comment:
I think we should use icons from [patternfly's icon list](https://www.patternfly.org/styles/icons/).  Perhaps `pficon-error-circle-o`?

Reply:
I thinks it's already a pf icon at `<MessageBox>`

Reply:
Oh I see.  I'm not a fan of how the `<MessageBox>` [restricts you](https://github.com/theforeman/foreman/blob/a5a6b5df465560729c86fa4799c1e9ad13941444/webpack/assets/javascripts/react_app/components/common/MessageBox.js#L11) to using `pficon-${iconType}` (as patternfly also recommends font awesome icons in some cases) but that certainly has nothing to do with this PR.